cranelift/filetests/filetests/isa/s390x/tls_elf.clif

```diff
@@ -37,7 +37,7 @@ block0(v0: i32):
 ;   .byte 0x00, 0x00
 ;   .byte 0x00, 0x00
 ;   lg %r2, 0(%r1)
-;   brasl %r14, 0x22 ; reloc_external TlsGdCall u1:0 0
+;   brasl %r14, 0x22 ; reloc_external PLTRel32Dbl %ElfTlsGetOffset 2
```

Comment:
Shouldn't both relocs be printed here?

Reply:
Good point.  I've fixed the disassembler to print multiple relocs on a single insn if present.